bps/templates/bps_linearizer.yaml

```diff
@@ -0,0 +1,11 @@
+pipelineYaml: ${CP_PIPE_DIR}/pipelines/${INSTRUMENT}/cpLinearizer.yaml
+
+project: ${TICKET}
+campaign: ${TICKET}
+
+payload:
+  payloadName: ${INSTRUMENT}_${TICKET}
+  output: ${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG}/linearizerGen.${RERUN}
+  butlerConfig: ${REPO}
+  inCollection: ${USER_CALIB_PREFIX}${INSTRUMENT}/calib/${TICKET}/${TAG},${RAW_COLLECTION},${CALIB_COLLECTIONS}
+  dataQuery: ${SELECTION_PTC_LINEARIZER-$SELECTION_PTC}
```

Comment:
This has a typo as well.

Reply:
More bash magic that is supposed to use `$SELECTION_PTC_LINEARIZER` if it is set and otherwise use `$SELECTION_PTC`.  Also not yet tested if the bps vars support this...